README.md

```diff
@@ -214,49 +210,104 @@ inputs = generate_random_inputs(onnx_filepath, batch_size)
 
 For a more in-depth read on available APIs and workflows, check out the [examples](https://github.com/neuralmagic/deepsparse/blob/main/examples/) and [DeepSparse Engine documentation.](https://docs.neuralmagic.com/deepsparse)
 
-## Resources
 
-### Learning More
 
-- Documentation: [SparseML,](https://docs.neuralmagic.com/sparseml/) [SparseZoo,](https://docs.neuralmagic.com/sparsezoo/) [Sparsify,](https://docs.neuralmagic.com/sparsify/) [DeepSparse](https://docs.neuralmagic.com/deepsparse/)
-- Neural Magic: [Blog,](https://www.neuralmagic.com/blog/) [Resources](https://www.neuralmagic.com/resources/)
+__ __
 
-### Release History
+## Scheduling Single-Stream, Multi-Stream, and Elastic Inference ⚡
 
-Official builds are hosted on PyPI
+The DeepSparse Engine offers up to 3 types of inferences based on your use-case. You can read more details here: [Inference Types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).
 
-- stable: [deepsparse](https://pypi.org/project/deepsparse)
-- nightly (dev): [deepsparse-nightly](https://pypi.org/project/deepsparse-nightly/)
+1 ⚡ Single stream scheduling: the latency/synchronous scenario, requests execute serially. [`default`]
 
-Additionally, more information can be found via [GitHub Releases.](https://github.com/neuralmagic/deepsparse/releases)
+<img src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/single-stream.png" alt="single stream diagram" />
 
-### License
+PRO TIP: It's highly optimized for minimum per-request latency, using all of the system's resources provided to it on every request it gets.
 
-The project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
+2 ⚡ Multi-stream scheduling: the throughput/asynchronous scenario, requests execute in parallel.
 
-Example files and scripts included in this repository are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
+<img src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/multi-stream.png" alt="multi stream diagram" />
+
+PRO TIP: The most common use cases for the multi-stream scheduler are where parallelism is low with respect to core count, and where requests need to be made asynchronously without time to batch them.
+
+3 ⚡ Elastic scheduling: requests execute in parallel, but not multiplexed on individual NUMA Nodes.
+
+PRO TIP: A workload that might benefit from the elastic scheduler is one in which multiple requests need to be handled simultaneously, but where performance is hindered when those requests have to share an L3 cache.
+__ __
+
+## Hardware Support 🧰
+
+The DeepSparse Engine is validated to work on x86 Intel and AMD CPUs running Linux operating systems. Mac and Windows require running Linux in a Docker or virtual machine.
```

Comment:
These two paragraphs got updated in the most recent version of readme.md

Reply:
updated paragraphs to match the most recent version thank you.